imap_processing/hit/l1a/hit_cdf_attrs.py

```diff
@@ -0,0 +1,762 @@
+        label_axis="5.1VD Ebox (Digital)",
+        units="V",
+        format="F1.2",
+    ),
+    "ebox_p12va": replace(
+        hit_hk_base_attrs,
+        validmin=11,  # doc says min 11.2
+        validmax=14,  # doc says max 13.1
+        catdesc="+12VA Ebox (Analog)",
+        fieldname="+12VA Ebox (Analog)",
+        label_axis="+12VA Ebox (Analog)",
+        units="V",
+        format="F2.1",
+    ),
+    "ebox_m12va": replace(
+        hit_hk_base_attrs,
+        validmin=-14,  # doc says min -13.1
+        validmax=-12,  # doc says max -11.2
+        catdesc="-12VA Ebox (Analog)",
+        fieldname="-12VA Ebox (Analog)",
+        label_axis="-12VA Ebox (Analog)",
+        units="V",
+        format="F2.1",
+    ),
+    "ebox_p5d7va": replace(
+        hit_hk_base_attrs,
+        validmin=5,  # doc says min 5.3
+        validmax=7,  # doc says max 6.3
+        catdesc="+5.7VA Ebox (Analog)",
+        fieldname="+5.7VA Ebox (Analog)",
+        label_axis="+5.7VA Ebox (Analog)",
+        units="V",
+        format="F1.1",
+    ),
+    "ebox_m5d7va": replace(
+        hit_hk_base_attrs,
+        validmin=-7,  # doc says min -6.4125
+        validmax=-5,  # doc says max -5.25
+        catdesc="-5.7VA Ebox (Analog)",
+        fieldname="-5.7VA Ebox (Analog)",
+        label_axis="-5.7VA Ebox (Analog)",
+        units="V",
+        format="F1.4",
+    ),
+    "ref_p5v": replace(
+        hit_hk_base_attrs,
+        # validmin=,             Need more info from instrument team
+        # validmax=,
+        catdesc="+5V ref",
+        fieldname="+5V ref",
+        label_axis="+5V ref",
+        units="V",
+        format="I4",
+    ),
+    "l1ab_bias": replace(
+        hit_hk_base_attrs,
+        validmin=0,
+        validmax=15,
+        catdesc="L1A/B Bias",
+        fieldname="L1A/B Bias",
+        label_axis="L1A/B Bias",
+        units="V",
+        format="I2",
+    ),
+    "l2ab_bias": replace(
+        hit_hk_base_attrs,
+        validmin=0,
+        validmax=25,
+        catdesc="L2A/B Bias",
+        fieldname="L2A/B Bias",
+        label_axis="L2A/B Bias",
+        units="V",
+        format="I2",
+    ),
+    "l34a_bias": replace(
+        hit_hk_base_attrs,
+        validmin=0,
+        validmax=255,
+        catdesc="L3/4A Bias",
+        fieldname="L3/4A Bias",
+        label_axis="L3/4A Bias",
+        units="V",
+        format="I3",
+    ),
+    "l34b_bias": replace(
+        hit_hk_base_attrs,
+        validmin=0,
+        validmax=225,
+        catdesc="L3/4B Bias",
+        fieldname="L3/4B Bias",
+        label_axis="L3/4B Bias",
+        units="V",
+        format="I3",
+    ),
+    "ebox_p2d0vd": replace(
+        hit_hk_base_attrs,
+        validmin=1,  # doc says min 1.79
+        validmax=3,  # doc says max 2.5
+        catdesc="+2.0VD Ebox (Digital)",
+        fieldname="+2.0VD Ebox (Digital)",
+        label_axis="+2.0VD Ebox (Digital)",
+        units="V",
+        format="F1.2",
```

Comment:
If I understand fortran formats correctly for floats, it is Fw.d where w=total number of digits in the output, d=number of digits to the right of the decimal point. So, for this value I think you want `"F3.2"`.

Reply:
This is very helpful! I haven't worked with fortran formats before. I thought the first number was the number of digits left of the decimal and the second number was the number of digits right of the decimal. I'll update these values